apps/files/src/components/FileEntry/FileEntryName.vue

```diff
@@ -30,7 +30,7 @@
 		<span class="files-list__row-name-text" dir="auto">
 			<!-- Keep the filename stuck to the extension to avoid whitespace rendering issues-->
 			<span class="files-list__row-name-" v-text="basename" />
-			<span class="files-list__row-name-ext" v-text="extension" />
+			<span v-show="userConfigStore.userConfig.show_files_extensions" class="files-list__row-name-ext" v-text="extension" />
```

Comment:
any reason to keep it in DOM? I do not think it is toggled very often so there is a (minimal) performance boost not to add it to the DOM.

```suggestion
			<span v-if="userConfigStore.userConfig.show_files_extensions" class="files-list__row-name-ext" v-text="extension" />
```